test/test_view.py

```diff
@@ -252,20 +252,11 @@ def test_ChoicesEditor(self):
             none_completion._to_dict(),
             CompletionValue(name_2, 'B')._to_dict()
         ])
-        editor.set_value(name_4)
```

Comment:
hebben we een equivalente alternatieve test op niveau vd delegate ?  want dit soort subtiliteiten durft al eens verloren te gaan ...

Reply:
Nee, eigenlijk hebben we een equivalent nodig dat via het model gaat